packages/scheduler/src/forks/SchedulerHostConfig.default.js

```diff
@@ -152,48 +152,6 @@ if (
   // We use the postMessage trick to defer idle work until after the repaint.
   const channel = new MessageChannel();
   const port = channel.port2;
-  channel.port1.onmessage = function(event) {
-    isMessageEventScheduled = false;
-
-    const prevScheduledCallback = scheduledHostCallback;
-    const prevTimeoutTime = timeoutTime;
-    scheduledHostCallback = null;
-    timeoutTime = -1;
-
-    const currentTime = getCurrentTime();
-
-    let didTimeout = false;
-    if (frameDeadline - currentTime <= 0) {
-      // There's no time left in this idle period. Check if the callback has
-      // a timeout and whether it's been exceeded.
-      if (prevTimeoutTime !== -1 && prevTimeoutTime <= currentTime) {
-        // Exceeded the timeout. Invoke the callback even though there's no
-        // time left.
-        didTimeout = true;
-      } else {
-        // No timeout.
-        if (!isAnimationFrameScheduled) {
-          // Schedule another animation callback so we retry later.
-          isAnimationFrameScheduled = true;
-          requestAnimationFrameWithTimeout(animationTick);
```

Comment:
there is no `animationTick` because we declare `animationTick` below(use `const`), the reason there is no throw is seems that the we will never enter the `if (!isAnimationFrameScheduled) {` statement(I don't know why), so just for now, I just move the whole function under the `animationTick  declaration`

Reply:
This is inside a clousre. `animationTick` is outside the closure. It will already be defined by the time the `onmessage` handler fires.

Reply:
Ah, yea, sorry, I didn't notice that